mysite/polls/admin.py

```diff
@@ -1,14 +1,61 @@
+from django import forms
 from django.contrib import admin
+# from django.contrib import messages
```

Comment:
Let's clean up comments like this.

Reply:
Got it. I left them in there while working just in case I needed to use Django messages, but I will delete them now that I don't need them.